region/norfair/east.json

```diff
@@ -2915,12 +2911,24 @@
               "id": 2,
               "requires": [
                 {"or": [
-                  "canSwim",
-                  "canSuitlessLavaDive"
+                  "heatProof",
+                  "canHeatRun"
+                ]},
+                {"or": [
+                  {"and": [
+                    "SpaceJump",
+                    "canSwim",
+                    {"cost": [
+                      {"heatFrames": 450}
+                    ]}
+                  ]},
+                  {"cost": [
+                    {"heatFrames": 600},
```

Comment:
I think this should have `canSuitlessLavaDive` as well?

Reply:
True